catalog/datasets/aat.jsonld

```diff
@@ -8,6 +8,11 @@
       "@value": "Art & Architecture Thesaurus (AAT)"
     }
   ],
+  "creator": [
```

Comment:
I used `schema:creator` here, but would be more than happy to discuss whether `schema:publisher` is more appropriate.

Reply:
Agree!